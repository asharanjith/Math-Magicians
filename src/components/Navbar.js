import React from 'react';
import { Link } from 'react-router-dom';

class Navbar extends React.Component {
  constructor(props) {
    super(props);
    this.state = { };
  }

  render() {
    return (
      <nav className="navbar">
        <h1 className="navbar__title">Math Magicians</h1>
        <ul className="navbar__links">
          <li><Link to="/" className="navbar__link">Home</Link></li>
          <li>
            {' '}
            <Link to="Calculator" className="navbar__link">Calculator</Link>
          </li>
          <li>
            {' '}
            <Link to="Quotes" className="navbar__link">Quotes</Link>
          </li>
        </ul>
      </nav>
    );
  }
}

export default Navbar;
