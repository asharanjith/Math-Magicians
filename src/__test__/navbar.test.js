import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Navbar from '../components/Navbar';

describe('check Navbar', () => {
  const tree = renderer.create(<Router><Navbar /></Router>).toJSON();
  test('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
