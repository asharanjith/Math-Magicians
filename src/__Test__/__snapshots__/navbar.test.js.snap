// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check Navbar renders correctly 1`] = `
<nav
  className="navbar"
>
  <h1
    className="navbar__title"
  >
    Math Magicians
  </h1>
  <ul
    className="navbar__links"
  >
    <li>
      <a
        className="navbar__link"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
       
      <a
        className="navbar__link"
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
       
      <a
        className="navbar__link"
        href="/Quotes"
        onClick={[Function]}
      >
        Quotes
      </a>
    </li>
  </ul>
</nav>
`;
